META.json

```diff
@@ -13,7 +13,7 @@
             "PostgreSQL": ">= 10.0"
          }
          "recommends": {
-            "PostgreSQL": "< 11"
+            "PostgreSQL": "< 14"
          }
```

Comment:
Looks like there is a comma missing on line 14

Reply:
Good shout. I never actually uploaded it to PGXN, so this file has not been in use.